go/api/command/command.proto

```diff
@@ -87,6 +87,17 @@ message VirtualInput {
   bool is_empty_directory = 4;
 }
 
+message SymlinkBehaviorType {
+  enum Value {
```

Comment:
Why do you need this nest?

Also follow
https://developers.google.com/protocol-buffers/docs/style#enums
?

Reply:
I followed how other enums are configured.  To be honest, I do not understand why this is needed.  I just tried to guess the coding style here from CommandResultStatus and InputType.